subprojects/gradle/runner/shared-test/src/main/kotlin/com/avito/runner/test/mock/MockDevice.kt

```diff
@@ -1,25 +1,34 @@
 package com.avito.runner.test.mock
 
 import com.avito.logger.Logger
+import com.avito.report.model.TestRuntimeData
+import com.avito.report.model.TestRuntimeDataPackage
 import com.avito.runner.service.model.DeviceTestCaseRun
 import com.avito.runner.service.model.TestCaseRun
 import com.avito.runner.service.model.intention.InstrumentationTestRunAction
 import com.avito.runner.service.worker.device.Device
 import com.avito.runner.service.worker.device.DeviceCoordinate
-import com.avito.runner.service.worker.device.Serial
 import com.avito.runner.service.worker.device.model.getData
 import com.avito.runner.service.worker.model.DeviceInstallation
 import com.avito.runner.service.worker.model.Installation
+import com.avito.runner.test.randomDeviceCoordinate
 import com.google.common.truth.Truth.assertWithMessage
+import com.google.gson.Gson
 import org.funktionale.tries.Try
 import java.io.File
 import java.nio.file.Path
 import java.util.ArrayDeque
+import java.util.Date
 import java.util.Queue
 
 open class MockDevice(
-    override val coordinate: DeviceCoordinate,
-    private val logger: Logger,
+    override val coordinate: DeviceCoordinate = randomDeviceCoordinate(),
+    private val logger: Logger = object : Logger {
```

Comment:
emptyLogger?

Reply:
not available here, i removed default value